lms/djangoapps/courseware/tests/test_split_module.py

```diff
@@ -123,7 +123,7 @@ def _check_split_test(self, user_tag):
         content = resp.content
 
         # Assert we see the proper icon in the top display
-        self.assertIn('<a class="{} inactive progress-0 nav-item"'.format(self.ICON_CLASSES[user_tag]), content)
+        self.assertIn('<button class="{} inactive progress-0 nav-item"'.format(self.ICON_CLASSES[user_tag]), content)
```

Comment:
We should have a wider discussion regarding our testing convention for this.  There are 2 different opinions on this.  1 side advocates your position, where it's better to keep test code robust against HTML changes.  However, the other side notes that it's Ok to update unit tests that specifically verify the HTML rendering whenever the HTML changes.

However, having said that, I'm of the opinion that testing of the HTML rendering for a sequence should be covered only by tests specifically for the sequence.  The fact that a test for the split_test module needed to be updated when the HTML for the sequence changed, is not ideal.